Builds failing due to new tifffile release

## Description

Example failure: https://travis-ci.org/scikit-image/scikit-image/jobs/651311082#L2729

```pytb
_____________________ ERROR at setup of test_imread_uint16 _____________________
    def setup():
>       use_plugin('tifffile')
../../venv/lib/python3.6/site-packages/skimage/io/tests/test_tifffile.py:14: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../venv/lib/python3.6/site-packages/skimage/io/manage_plugins.py:253: in use_plugin
    _load(name)
../../venv/lib/python3.6/site-packages/skimage/io/manage_plugins.py:297: in _load
    fromlist=[modname])
../../venv/lib/python3.6/site-packages/skimage/io/_plugins/tifffile_plugin.py:4: in <module>
    from tifffile import TiffFile, imsave, parse_kwargs
../../venv/lib/python3.6/site-packages/tifffile/__init__.py:3: in <module>
    from .tifffile import __doc__, __all__, __version__, main
../../venv/lib/python3.6/site-packages/tifffile/tifffile.py:524: in <module>
    import imagecodecs
../../venv/lib/python3.6/site-packages/imagecodecs/__init__.py:5: in <module>
    from .imagecodecs import __doc__, __version__, __getattr__, __dir__
../../venv/lib/python3.6/site-packages/imagecodecs/imagecodecs.py:804: in <module>
    _load_all()
../../venv/lib/python3.6/site-packages/imagecodecs/imagecodecs.py:347: in _load_all
    __getattr__(name)
../../venv/lib/python3.6/site-packages/imagecodecs/imagecodecs.py:371: in __getattr__
    module = importlib.import_module('._' + module_, 'imagecodecs')
../../venv/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
>   ???
E   AttributeError: type object 'imagecodecs._aec.array' has no attribute '__reduce_cython__'
```

I forget where the `has no attribute '__reduce_cython__'` error comes from but we've seen it before...

--- a/requirements/optional.txt
+++ b/requirements/optional.txt
@@ -1,6 +1,6 @@
 SimpleITK
 astropy>=1.2.0
-tifffile
+tifffile>=0.11.1
 qtpy
 pyamg
 dask[array]>=0.15.0

